README.md

```diff
@@ -20,12 +20,13 @@
 Would you like to learn about Rhino hands-on? Join our events!
 Upcoming events will be added to this section - stay tuned.
 
+<details>
+<summary><h3>Past</h3> <i>(click to expand)</i></summary>
```

Comment:
I think it would look better if it were inlined (on the documentation page, README on GitHub looks good).
![image](https://github.com/Appsilon/rhino/assets/7279715/d5e2f7b6-de7f-4b4f-86b3-8d2b8dcf7400)

Reply:
Good catch! Fixed in 0cf94346fb698cf4cdfdd6c28181dbd9e3378652, tested in Chromium, Safari, and Firefox.